frog/imports/client/Wiki/WikiTopNavbar.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
```

Comment:
What happened to this file - is this because of a merge conflict?

Reply:
Yes, somehow git merge re-introduced this file

Reply:
(Removed now)